src/context/WalletProvider/NativeWallet/components/NativeTestPhrase.tsx

```diff
@@ -111,7 +113,11 @@ export const NativeTestPhrase = ({ history, location }: NativeSetupProps) => {
             translation={'walletProvider.shapeShift.testPhrase.body'}
           />{' '}
           <Tag colorScheme='green'>
-            {`${testState.targetWordIndex + 1}${ordinalSuffix(testState.targetWordIndex + 1)}`}
+            {translate(
+              `walletProvider.shapeShift.testPhrase.${testState.targetWordIndex + 1}${ordinalSuffix(
+                testState.targetWordIndex + 1,
+              )}`,
```

Comment:
You could pull `testState.targetWordIndex + 1` into `targetWordNumber` or something to reduce the duplication, but it's fine.